dataverse/orgsvc/C#-NETCore/xMultipleSamples/Utility.cs

```diff
@@ -276,34 +275,44 @@ public static void CreateExampleTable(IOrganizationService service, string table
                 return;
             }
 
-            Console.WriteLine($"Creating {tableSchemaName} table...");
+            string tableType = isElastic ? "Elastic" : "Standard";
+            Console.WriteLine($"Creating {tableSchemaName} table of TableType {tableType}..");
 
-            CreateEntityRequest createEntityRequest = new()
+            if (isElastic)
             {
-                Entity = new EntityMetadata
-                {
-                    SchemaName = tableSchemaName,
-                    DisplayName = new Label("Example", 1033),
-                    DisplayCollectionName = new Label("Examples", 1033),
-                    Description = new Label("A table for code samples.", 1033),
-                    OwnershipType = OwnershipTypes.UserOwned,
-                    IsActivity = false
-
-                },
-                PrimaryAttribute = new StringAttributeMetadata
+                ElasticUtility.CreateExampleTable(service, tableSchemaName);
+            }
+            else
+            {
+                CreateEntityRequest createEntityRequest = new()
                 {
-                    SchemaName = "sample_Name",
-                    RequiredLevel = new AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
-                    MaxLength = 100,
-                    FormatName = StringFormatName.Text,
-                    DisplayName = new Label("Example Name", 1033),
-                    Description = new Label("The name of the example record.", 1033)
-                },
-                HasActivities = true
-
-            };
+                    Entity = new EntityMetadata
+                    {
+                        SchemaName = tableSchemaName,
+                        DisplayName = new Label("Example", 1033),
+                        DisplayCollectionName = new Label("Examples", 1033),
+                        Description = new Label("A table for code samples.", 1033),
+                        OwnershipType = OwnershipTypes.UserOwned,
+                        IsActivity = false,
+                        DataProviderId = tableType == "Elastic" ? new Guid("1d9bde74-9ebd-4da9-8ff5-aa74945b9f74") : null,
```

Comment:
Do we want customers to set this specific GUID for DataProviderId?
I didn't see this mentioned in the docs..

Reply:
missed reverting this